Editor/Window/GameLiftPlugin.cs

```diff
@@ -13,7 +13,7 @@ namespace Editor.Window
 {
     public class GameLiftPlugin : UnityEditor.EditorWindow
     {
-        [SerializeField] private Texture _icon;
+        [SerializeField] internal Texture _icon;
```

Comment:
Please use a getter property instead of making this internal. Doing so exposes this property to being assigned via script which isn't desirable

Reply:
Created get property